DEVELOPMENT.md

```diff
@@ -93,25 +110,34 @@ After which, you can rerun a `px auth login` to authenticate with the dev cloud
 Make sure to `px delete --clobber` if running a prior instance of Vizier pointing to another cloud instance.
 
 ## Vizier
+Vizier is Pixie’s data collector that runs on each cluster. It is responsible for query execution and managing PEMs.
+
+### Getting Started
+Verify that you can build Pixie Vizier and run the unit tests.
+
+The following will build Pixie Vizier and run a unit test of your choice in Pixie's Vizier module. If you are running the development environment via Docker, the following should be run inside the Docker container.
 
-Deploying a development version of Vizier is a 2-step process. A release-version of Vizier must first be deployed (through the CLI or YAMLs), which will set up specific cluster-secrets that are not deployed via Skaffold. Skaffold can then be run to build and deploy a version of Vizier based on local changes.
+```
+bazel test //src/<path to unit test file> --test_output=errors -j $(nproc)
+```
+
+### Deploying
+Deploying a development version of Vizier is a 2-step process. An official release-version of Vizier must first be deployed (through the Pixie CLI or YAMLs) and Skaffold can then be run to build and deploy a local development version of Vizier.
 
-1. If deploying Vizier pointing to a dev cloud instance, make sure to export the following environment variables:
+1. If you wish to test development changes made to Pixie Cloud and Vizier, export the following environment variables that will point to the development Pixie Cloud instance:
```

Comment:
might want to add "both" here just so someone doesn't take accidentally take it as a logical OR